src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,52 @@
+package duke.command;
+
+import duke.task.Task;
+import duke.util.DukeException;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * Represents a delete command to delete a task from the list.
+ */
+public class DeleteCommand extends Command {
+    private final int taskNum;
+
+    /**
+     * Class constructor.
+     *
+     * @param taskNum The index of the task to be deleted.
+     */
+    public DeleteCommand(int taskNum) {
+        this.taskNum = taskNum;
+    }
+
+    /**
+     * Execute the delete command, print confirmation message and saves the updated task list to file.
```

Comment:
I think it should be executes, prints and saves, and I think it was repeated in several other java-docs you wrote for `Command::execute`